Sources/AppStoreConnectCLI/Commands/TestFlight/PreReleaseVersion/TestFlightPreReleaseVersionCommand.swift

```diff
@@ -6,9 +6,10 @@ import Foundation
 public struct TestFlightPreReleaseVersionCommand: ParsableCommand {
     public static var configuration = CommandConfiguration(
         commandName: "prereleaseversion",
-        abstract: "Platform-specific versions of your app intended for distribution to beta testers.",
+        abstract: "PreRelease version commands.",
```

Comment:
I think the original description here was more explicit. It was taken directly from the apple documentation
https://developer.apple.com/documentation/appstoreconnectapi/testflight/prerelease_versions